.github/workflows/linux.yml

```diff
@@ -253,6 +253,22 @@ jobs:
         cmake --build build --parallel 2
         ctest --test-dir build --output-on-failure
 
+        python3 -m pip install jsonschema
+        cd share/openPMD/json_schema
+        PATH="../../../build/bin:$PATH" make -j 2
+        # We need to exclude the thetaMode example since that has a different
+        # meshesPath and the JSON schema needs to hardcode that.
```

Comment:
Are we able to patch this in `check.py`?

Reply:
Not very easily. The JSON schema is on the file system and the single .json files refer to each other by their file names. Changing this would require (1) traversing the entire JSON schema and overriding the meshes path, the particles path and the references and (2) somehow setting up python-jsonschema to cross-reference in-memory schemas which I don't even know if it supports that, both at runtime of `check.py`.